ztp/kube-compare-reference/optional/ptp-config/PtpConfigMasterForEvent.yaml

```diff
@@ -26,9 +26,9 @@ spec:
       #
       twoStepFlag 1
       slaveOnly 0
-      priority1 128
-      priority2 128
-      domainNumber 24
+      priority1 (?<priority1>[0-9]+)
+      priority2 (?<priority2>[0-9]+)
+      domainNumber (?<domainNumber>[0-9]+)
```

Comment:
since line 17 overrides line 31 do we still want to do it for readability ?

Reply:
Line 17 is for phc2sys, and line 31 is for ptp4l - Does line 17 affect line 31?